<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockSliceViewProperties</class>
 <widget class="QWidget" name="BlockSliceViewProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GridViewProperties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="optionsTree">
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>180</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Data</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Style</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Slice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSlice">
       <item>
        <property name="text">
         <string>X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>at</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnSlice">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
